### PR TITLE
GAT-1577: Running Unit Tests

### PR DESCRIPTION
**Pull request by kymmeh**

### CODE REVIEW

File: cloudbuild_dynamic.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ steps:
           [
               'run',
               'deploy',
-              '${_ENVIRONMENT}-web',
+              '${_ENVIRONMENT}-medicines-dashboard',
```
Comment (kymmeh) [inline cloudbuild_dynamic.yaml:22]: @reubensamuel is this change intentional?

Comment (reubensamuel) [reply cloudbuild_dynamic.yaml]: Definitely not perks of working on multiple projects :(

**Review by dannitahdruk — APPROVED** [review]

**Review by ThinkingMonkeyLtd — APPROVED** [review]
